lib/report_formatter/timeline.rb

```diff
@@ -236,6 +237,8 @@ def tl_event(tl_xml, row, col)
           if ems_cloud
             # restful route is used for cloud provider unlike infrastructure provider
             val = "&lt;a href='/ems_cloud/#{provider_id}'&gt;#{row[co]}&lt;/a&gt;"
+          elsif ems_container
+            val = "&lt;a href='/ems_container/show/#{provider_id}'&gt;#{row[co]}&lt;/a&gt;"
```

Comment:
@moolitayer i think you should compress the provider_id here just like ems_infra link below to be consistent or you could have just done ` val = "&lt;a href='/#{ems_container ? "ems_container" : "ems_infra"}/show/#{to_cid(provider_id)}'&gt;#{row[co]}&lt;/a&gt;" `